test/scripts/steps-lib.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+SCRIPT_NAME="steps-lib.sh"
+SCRIPT="$BATS_TEST_DIRNAME/../../ci/steps/$SCRIPT_NAME"
+
+source "$SCRIPT"
+
+@test "is_env_var_set should return 1 if env var is not set" {
+  run is_env_var_set "ASDF_TEST_SET"
+  [ "$output" = 1 ]
```

Comment:
If you change the functions to return stuff instead of echo, then you'd have to change this to check the `$status` instead of `$output`

Reply:
Good point! Okay if/after I refactor, I'll update these!